### PR TITLE
Update email templates for global partners

### PR DESCRIPTION
**Pull request by amakarudze**
Changes in this PR
- Update global partner email templates with the link to the new sponsor deck.
- Change the year to 2024 in sponsor email templates.
- Add slack invite link for new user  email.

### CODE REVIEW

**Comment by amakarudze** [comment]
Hey @marksweb, could you take a look at this for me? Thanks

File: templates/emails/globalpartners/prospective_sponsor.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 <p>We are currently looking for new corporate sponsors who will sponsor us through either monthly donations 
     or with a once off annual donation. I would like to check with you if your company would consider sponsoring Django Girls.</p>
 
-<p>Please see attached our sponsors deck for 2023 https://drive.google.com/file/d/1T2jL9tnu5ZkNvkQRAxd7jlt_BnX7VCLE/view 
+<p>Please see attached our sponsors deck for 2024 https://drive.google.com/file/d/1fAmyAiEyLGiE0s6_z3PzgLByHNKzEVHB/view
```
Comment (marksweb) [inline templates/emails/globalpartners/prospective_sponsor.html:10]: @amakarudze Do these dates always want to be the current year?

To dynamically display the current year you should be able to use [`{% now "Y" %}`](https://docs.djangoproject.com/en/dev/ref/templates/builtins/#now).

**Review by marksweb — CHANGES_REQUESTED** [review]

File: globalpartners/management/commands/send_sponsor_thank_you_email.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+class Command(BaseCommand):
+    help = "Sends thank you emails to sponsors at the end of the year"
+
+    def add_arguments(self, parser):
+        pass
+
+    def handle(self, *args, **kwargs):
+        today = date.today()
+        year = date.year()
+        if today == f"12/12/{year}":
```
[the remark this answers is not in the document]
Comment (amakarudze) [reply globalpartners/management/commands/send_sponsor_thank_you_email.py:18]: Oh, thanks for that. I wasn't aware I pushed this already. Let me fix it and work on tests for it. This part wasn't supposed to be deployed now. Let me complete everything then. I had just planned to deploy the changes in the email templates lol.

 Thanks @marksweb!

[the remark this answers is not in the document]
Comment (amakarudze) [reply globalpartners/management/commands/send_sponsor_thank_you_email.py]: Have fixed all the management commands and added tests now @marksweb.

**Review by marksweb — APPROVED** [review]

**Pull request merged**